discovery/src/main/java/com/ibm/watson/discovery/v1/Discovery.java

```diff
@@ -674,9 +699,7 @@ public Discovery(String versionDate, Authenticator authenticator) {
     }
     builder.header("Accept", "application/json");
     final JsonObject contentJson = new JsonObject();
-    if (updateCollectionOptions.name() != null) {
-      contentJson.addProperty("name", updateCollectionOptions.name());
-    }
+    contentJson.addProperty("name", updateCollectionOptions.name());
```

Comment:
Is this breaking?

Reply:
Nope, this change fell under the category of a parameter that's now required because the service always required it and would've thrown an error anyway.